k4LCIOReader/CMakeLists.txt

```diff
@@ -19,7 +19,9 @@ if(BUILD_TESTING)
     target_include_directories(k4lcioreader_read-lcio PUBLIC
         ${podio_INCLUDE_DIR})
     target_link_libraries(k4lcioreader_read-lcio k4LCIOReader)
-    add_test(NAME read-lcio COMMAND wget https://key4hep.web.cern.ch/key4hep/testFiles/ddsimOutput/testSimulation.slcio; k4lcioreader_read-lcio testSimulation.slcio)
+    add_test(NAME prepare-input-files COMMAND wget https://key4hep.web.cern.ch/key4hep/testFiles/ddsimOutput/testSimulation.slcio)
```

Comment:
Add a custom target that downloads the file and make the test depend on the existence of the file?
Make the executable target depend on the download target? Then re-running tests shouldn't re-download the file

Reply:
How about just running `wget -N`? and changing the DEPENDS to FIXTURES_REQUIRED for correctness

Reply:
that would still rerun wget, but less changes, so fine.